app/views/repair_requests/created.html.erb

```diff
@@ -20,7 +20,12 @@
           <% if @repair_request.work_orders.first.is_dlo? %>
             <%= link_to "open DRS", drs_url, target: "_blank" %> to book an appointment!
           <% else %>
-            open UH to book an appointment!
+            open UH
```

Comment:
I don't think this is what this story is asking.  If it's Immediate or Emergency priority then they should be opening DRS to check if the appointment has gone through, but not book it. They shouldn't be going to UH to book appointments.

Reply:
Should be good now!